CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 
 - Crashes when parsing invalid lockfiles
+- `SPDX` parsing adding project as a package dependency and error with certain
+    `SPDX` formatted text files having optional package fields
```

Comment:
I know this is still a draft but since I've already seen it: We shouldn't have changelog entries that contain multiple changes. So instead of an `and` in the entry, I recommend just putting it as a separate entry.